third_party/lua/README-luadebug.md

```diff
@@ -83,7 +89,39 @@ If you have used other CLI debuggers, debugger.lua shouldn't be surprising. I di
     w|where $linecount
     - print source code around the current line
 
-If you've never used a command line debugger before, start a nice warm cozy fire, run tutorial.lua, and open it up in your favorite editor so you can follow along.
+If you've never used a command line debugger before, start a nice warm cozy fire, run luadebug_tutorial.lua, and open it up in your favorite editor so you can follow along.
+
+Breakpoints
+-
+
+One could set breakpoint either using full syntax as:
+
+  break debug-target.lua:9
+
+Or using "short" syntax using only line numbers in a form `+NN` or `:NN`. In this
+case debugger will use current active script file as a basic for such breakpoint.
```

Comment:
> case debugger will use current active script file as a basic for such breakpoint.

It's not clear to me what `basic` means hear. Did you mean to use `base`.?

Reply:
Indeed

Reply:
Fixed